app/coffee/RedisManager.coffee

```diff
@@ -154,11 +155,11 @@ module.exports = RedisManager =
 					logger.error project_id: project_id, doc_id: doc_id, doc_project_id: doc_project_id, "doc missing from docsInProject set"
 				callback null, docLines, version, ranges, pathname, projectHistoryId, unflushedTime, lastUpdatedAt, lastUpdatedBy
 
-	getDocVersion: (doc_id, callback = (error, version) ->) ->
-		rclient.get keys.docVersion(doc_id: doc_id), (error, version) ->
+	getDocVersion: (doc_id, callback = (error, version, projectHistoryType) ->) ->
+		rclient.mget keys.docVersion(doc_id: doc_id), keys.projectHistoryType(doc_id:doc_id), (error, version, projectHistoryType) ->
```

Comment:
the redis [`MGET`](https://redis.io/commands/mget) command never fails since it will just return `nil` instead if it doesn't find anything. I haven't found any documentation about what this function does, but I'm wondering if we still need the error handling?

Reply:
Generally for the redis commands the errors from the command itself do come in the `result` but a genuine `error` can be returned in other cases, e.g. if there is a problem talking to redis.